db/migrate/20160329192055_add_virtual_instance_id_to_configured_systems.rb

```diff
@@ -0,0 +1,5 @@
+class AddVirtualInstanceIdToConfiguredSystems < ActiveRecord::Migration[5.0]
+  def change
+    add_column :configured_systems, :virtual_instance_id, :string
```

Comment:
This feels wrong to have a column name ending with `_id` that isn't a foreign key reference (i.e. bigint).  On that note, `host.instance_id` is a bad name for the same reason.